.rubocop.yml

```diff
@@ -157,6 +157,10 @@ Style/TrailingCommaInArrayLiteral:
 Rails/FilePath:
   EnforcedStyle: slashes
 
+Rails/UniqueValidationWithoutIndex:
```

Comment:
this change can be removed as now in master is disabled: 

https://github.com/thepracticaldev/dev.to/blob/e35d55c422a6e6e122800bf19eb643b1015fcd3a/.rubocop_todo.yml#L64-L65